### PR TITLE
[v1.x] Add more CV models to onnxruntime inference test, add bert model test.

### PR DESCRIPTION
**Pull request by josephevans**
## Description ##
Add BERT model test with onnxruntime, add additionally supported CV models to unit tests.

### CODE REVIEW

File: tests/python-pytest/onnx/test_onnxruntime.py [hunk 1]
```diff
@@ -118,4 +121,52 @@ def download_test_images(tmpdir):
...
+    ses_opt = onnxruntime.SessionOptions()
+    ses_opt.log_severity_level = 3
+    session = onnxruntime.InferenceSession(onnx_file, ses_opt)
+    onnx_inputs = [inputs, token_types, valid_length]
+    input_dict = dict((session.get_inputs()[i].name, onnx_inputs[i].asnumpy()) for i in range(len(onnx_inputs)))
+    pred_onx = session.run(None, input_dict)[0]
+
+    assert_almost_equal(seq_encoding, pred_onx)
+
+    shutil.rmtree(tmp_path)
```
Comment (marcoabreu) [inline tests/python-pytest/onnx/test_onnxruntime.py:170]: Use try finally

File: tests/python-pytest/onnx/test_onnxruntime.py [hunk 2]
```diff
@@ -97,25 +102,79 @@ def download_test_images(tmpdir):
...
+
+        # create onnxruntime session using the generated onnx file
+        ses_opt = onnxruntime.SessionOptions()
+        ses_opt.log_severity_level = 3
+        session = onnxruntime.InferenceSession(onnx_file, ses_opt)
+        onnx_inputs = [inputs, token_types, valid_length]
+        input_dict = dict((session.get_inputs()[i].name, onnx_inputs[i].asnumpy()) for i in range(len(onnx_inputs)))
+        pred_onx = session.run(None, input_dict)[0]
+
+        assert_almost_equal(seq_encoding, pred_onx)
```
Comment (Zha0q1) [inline tests/python-pytest/onnx/test_onnxruntime.py:175]: shall we also add
``` python
assert_almost_equal(cls_encoding, session.run(None, input_dict)[1])
```

**Review by waytrue17 — APPROVED** [review]
LGTM, thanks!

File: tests/python-pytest/onnx/test_onnxruntime.py (hunk 2 above)
Comment (josephevans) [reply tests/python-pytest/onnx/test_onnxruntime.py:175]: Added!

**Review by Zha0q1 — APPROVED** [review]
lgtm!

**Pull request merged**